### ISSUE TITLE
nrsysmond-2.0.0.105 in 2014Q3 has an incorrect logfile string in default config.

### ISSUE THREAD

**Issue by baetheus** [TritonDataCenter/pkgsrc-legacy#236]
Issue:
Default configuration file does not correctly reference /var/log. This causes the service to fall into maintenance. The error is obviously not logged until the binary is run with the -f flag to show errors in the foreground.

To Reproduce:
From within a base64 14.3.0 zone.

```
pkgin -fy up
pkgin fug
pkgin in nrsysmond
grep logfile= /opt/local/etc/nrsysmond.cfg 
```

The result of grep will be:

```
logfile=@VARBASE@/newrelic/nrsysmond.log
```

But should be:

```
logfile=/var/log/newrelic/nrsysmond.log
```

**Comment by mamash** [comment]
Thanks Brandon, I'll handle ASAP.

**Comment by mamash** [comment]
Fixed as part of the update to 2.0.2.111 last night.
